### DISCUSSION TITLE
Looking for an example for using `ValueTransformation`

### DISCUSSION THREAD

**Discussion by barvhaim** [SigmaHQ/pySigma#91] [Q&A] [answered]
Hi, I am looking for an example for how to use `ValueTransformation` but I want to apply this transformation only for a specific fields, is the value transformation have a context of a field? for example `cast to string value if field is eventCode`. is it possible?

**Comment by barvhaim** [comment]
I have added a custom transformer class into the pipeline, is that the correct way? https://github.com/barvhaim/pySigma-backend-stix/blob/master/sigma/pipelines/stix/stix.py#L552

**Comment by thomaspatzke** [comment, marked as answer]
Looks goof for me! An example is the [ReplaceStringTransformation](https://github.com/SigmaHQ/pySigma/blob/2b4a78ab1900dcbe9c59b2aad213d4e9e2892c08/sigma/processing/transformations.py#L449).
